### PR TITLE
Add Bird, Tier, and Voi

### PR DESCRIPTION
**Pull request by mal-tee**

### CODE REVIEW

File: bird-rides-europe.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    "slug": "bird-rides-europe",
+    "relevant-countries": [
+        "all"
+    ],
+    "name": "Bird Rides Europe, B.V.",
+    "runs": [
+        "Circ (formerly)"
+    ],
+    "address": "Weesperstraat 61\n1018 VN Amsterdam\nNetherlands",
+    "email": "hello@bird.co",
```
Comment (baltpeter) [inline bird-rides-europe.json:11]: Please add the website.

File: tier-mobility.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+        "fi",
+        "fr",
+        "no",
+        "ch",
+        "uk"
+    ],
+    "name": "Tier Mobility GmbH",
+    "address": "c/o DataCo GmbH\nDachauer Str. 65\n80335 München\nDeutschland",
+    "email": "privacy@tier.app",
+    "web": "www.tier.app",
```
Comment (baltpeter) [inline tier-mobility.json:15]:
```suggestion
    "web": "https://www.tier.app",
```

**Review by baltpeter — CHANGES_REQUESTED** [review]
Thank you!

File: bird-rides-europe.json (hunk 1 above)
Comment (mal-tee) [reply bird-rides-europe.json:11]: Oops, done.

**Review by baltpeter — APPROVED** [review]

**Pull request merged**